docs/ops/arithmetic/Sin_1.md

```diff
@@ -6,13 +6,20 @@
 
 **Short description**: *Sin* performs element-wise sine operation with given tensor.
 
+**Detailed description**: *sin* does the following with the input tensor *a*:
+\f[
+a_{i} = sin(a_{i})
+\f]
+
+The best description of the sine operation can be found on [Wikipedia](https://en.wikipedia.org/wiki/Sine)
```

Comment:
Let's remove this line. The sine operation is obvious. But we should specify what is the input to the function: radians or degrees?